### ISSUE TITLE
Why isn't the footer content hardcoded?

### ISSUE THREAD

**Issue by ashooner** [uky-web-archive/seeblue#31]
I recall this change being made, but I'm not sure I ever completely understood why. Is there a reason the global footer (uk logo, seeblue logo, and copyright statement) are using PHP in a block rather than just being included in the template? If it was the text-only link, isn't that being phased out anyway?

**Comment by petervcook** [comment]
I found this note in GitHub: https://github.com/UK-PR/seeblue/issues/14

And an email from myself on November 6th: "since we have multiple page templates in the theme, we moved the copyright notice in the footer to a block contained within another region"

I am sure it made good sense at the time ;-)

**Comment by petervcook** [comment]
Andy, just want to follow up here. Originally it was hard coded, but there was a decision back in November to un-hardcode it. 
Do we need to re-hardcode it?

**Comment by cornettims** [comment]
We updated the page templates to include the footer markup/logic (in the 201406-dev branch).

Note that all we did was copy the content into the page template. Should we remove the region the block was in or leave it there? If we don't remove the region and the updates are pushed to a production site, they'll have duplicate content in the footer until the block is unassigned from the drupal backend. Is there any reason to keep that region? Looks like that's the only block being output there so it may be feasible/easier to  just remove the region from the theme completely.
